storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -74,6 +74,9 @@ CREATE TABLE oasis_3.events
   FOREIGN KEY (tx_block, tx_index) REFERENCES oasis_3.transactions(block, tx_index) DEFERRABLE INITIALLY DEFERRED
 );
 CREATE INDEX ix_events_related_accounts ON oasis_3.events USING gin(related_accounts);
+CREATE INDEX ix_events_tx_block ON oasis_3.events (tx_block);  -- for fetching events without filters
```

Comment:
should the index be appended too? eg, `(tx_block, tx_index)`

Reply:
theory was that they can seek to the block and then walk through a sane number of events. should see if any query plans go haywire